hw4.Rmd

```diff
@@ -30,43 +37,72 @@ Before doing anything else, just play with the data to get a sense of it.
 (@) The main outcome of interest is *decision* which records whether the judge granted an appeal or not.  In what proprotion of cases did the judge grant an appeal.
 
 ```{r}
-# [your code here]
+#### Odds and log-odds #####
+df %>%
+  summarise(count(df$decision)) %>%
+  summarise(prob.appeal = mean(df$decision==1))
+# probability of decison= 0.296875
+
 ```
 
 (@) There are 12 different judges in the data. A key question is whether different judges have different rates of granting an appeal.  Make a plot showing the number of cases per judge.
 
 ```{r}
-# [your code here]
+ggplot(df, aes(x = judge, fill = judge)) + 
+  stat_bin(type = "identity", width = 0.60) + 
+  labs(title = "Amount of Appeals per Judge", x = "Judges", y = "Appeals") +
+  coord_flip() 
+  
 ```
 
 (@) Now plot the rate at which the judges grant appeals. 
 
 ```{r}
-# [your code here]
+df <- df %>%
+  group_by(judge) %>%
```

Comment:
I think you are having trouble because of how you assigned your data frame after summarising count in line 42. And instead of using sum/length you could have just used mean.